### PR TITLE
BTA-1546: Create ADD VAT Pages

### PR DESCRIPTION
**Pull request by nehakannaujia1**
BTA-1546: Create ADD VAT Pages

### CODE REVIEW

**Comment by suyangyu** [comment]
added do not merge pending feature toggle

File: conf/app.routes [hunk 1]
```diff
@@ -305,3 +305,29 @@ GET        /employer/have-paye-ref                      controllers.employer.pay
 POST       /employer/have-paye-ref                      controllers.employer.paye.DoYouHavePAYEReferenceController.onSubmit()
 
 GET        /corporation-tax                             controllers.ct.IndividualAddCorporationTaxController.onPageLoad()
+
+GET        /vat/registration-process                    controllers.vat.VatRegistrationProcessController.onPageLoad()
+
+GET        /vat/eligibility                             controllers.vat.VatEligibilityCheckController.onPageLoad()
+
+GET        /vat/vat-exception                           controllers.vat.VatRegistrationExceptionController.onPageLoad()
```
Comment (chris-ianson) [inline conf/app.routes]: Could we change _vat-exception_ to _has-vat-exception_ just to keep the routes consistent, please?

File: conf/messages.cy [hunk 2]
```diff
@@ -1230,3 +1230,98 @@ individualAddCorporationTax.p1 = Mae hwn yn gyfrif ''Unigolyn
...
+#######################################################
+##  VATRegistrationProcess
+#######################################################
+vatRegistrationProcess.title = Dechrau’r broses gofrestru ar gyfer TAW
+vatRegistrationProcess.heading = Dechrau’r broses gofrestru ar gyfer TAW
+vatRegistrationProcess.continue = Dechrau’r broses gofrestru ar gyfer TAW
+vatRegistrationProcess.listitem1 = Check if you can register for VAT using the online service
+vatRegistrationProcess.listitem2 = Complete VAT registration
+vatRegistrationProcess.listitem3 = Get your VAT registration number by post or email reminder
+vatRegistrationProcess.listitem4 = Sign into your business tax account to get online access to VAT
```
Comment (chris-ianson) [inline conf/messages.cy]: This and lines above it should be welsh content.

File: test/controllers/vat/ClaimRefundControllerSpec.scala [hunk 3]
```diff
@@ -1,4 +1,20 @@
-package controllers.sa
```
Comment (chris-ianson) [inline test/controllers/vat/ClaimRefundControllerSpec.scala]: I don't think this should be here. generated-test directories are removed after running migration.  Do we need it?

File: test/views/vat/AgriculturalFlatRateSchemeViewSpec.scala [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+  val form = new AgriculturalFlatRateSchemeFormProvider()()
+
+  val serviceInfoContent = HtmlFormat.empty
+
+  def createView = () => agriculturalFlatRateScheme(frontendAppConfig, form)(serviceInfoContent)(fakeRequest, messages)
+
+  def createViewUsingForm =
+    (form: Form[_]) => agriculturalFlatRateScheme(frontendAppConfig, form)(serviceInfoContent)(fakeRequest, messages)
+
+  def viewIncludes(s: String): Unit = asDocument(createView()).text() must include(s)
```
Comment (chris-ianson) [inline test/views/vat/AgriculturalFlatRateSchemeViewSpec.scala]: Consider removing this method which is duplicated across the ViewSpecs and using _assertContainsText_ method which is part of _ViewSpecBase_

Comment (nehakannaujia1) [reply test/views/vat/AgriculturalFlatRateSchemeViewSpec.scala]: Done

File: test/controllers/vat/ClaimRefundControllerSpec.scala (hunk 3 above)
Comment (nehakannaujia1) [reply test/controllers/vat/ClaimRefundControllerSpec.scala]: couldnot find this.

File: conf/messages.cy (hunk 2 above)
Comment (nehakannaujia1) [reply conf/messages.cy]: done.

File: conf/app.routes (hunk 1 above)
Comment (nehakannaujia1) [reply conf/app.routes]: done

**Pull request closed without merging**